packages/grafana-ui/src/components/Table/TableCell.tsx

```diff
@@ -32,15 +41,33 @@ export const TableCell: FC<Props> = ({ cell, field, tableStyles, onCellFilterAdd
     innerWidth -= tableStyles.lastChildExtraPadding;
   }
 
-  return (
-    <>
-      {cell.render('Cell', {
-        field,
-        tableStyles,
-        onCellFilterAdded,
-        cellProps,
-        innerWidth,
-      })}
-    </>
+  const link: LinkModel<any> | undefined = field.getLinks?.({
```

Comment:
nit: no need for any
```suggestion
  const link: LinkModel | undefined = field.getLinks?.({
```